reference/3.0/Microsoft.PowerShell.Core/About/about_Execution_Policies.md

```diff
@@ -254,8 +266,8 @@ For example:
 powershell.exe -ExecutionPolicy AllSigned
 ```
 
-The execution policy that you set is not stored in the registry. Instead, it
-is stored in the `$env:PSExecutionPolicyPreference` environment variable. The
+The execution policy that you set is not stored in the registry. Instead, it is
```

Comment:
Use the contraction form of 'is not' and 'it is'.

Reply:
Resolved, thanks for the feedback.